PayPalWebPayments/src/main/java/com/paypal/android/paypalwebpayments/PayPalWebCheckoutClient.kt

```diff
@@ -18,9 +24,14 @@ import com.paypal.android.paypalwebpayments.analytics.VaultEvent
 class PayPalWebCheckoutClient internal constructor(
     private val analytics: PayPalWebAnalytics,
     private val payPalWebLauncher: PayPalWebLauncher,
-    private val sessionStore: PayPalWebCheckoutSessionStore = PayPalWebCheckoutSessionStore()
+    private val sessionStore: PayPalWebCheckoutSessionStore,
+    private val updateClientConfigAPI: UpdateClientConfigAPI,
+    private val ioDispatcher: CoroutineDispatcher
 ) {
 
+    // application scope to execute fire-and-forget tasks
```

Comment:
```suggestion
    // application scope to execute CCO updates
```
Since we're no longer using fire and forget.